sample/simple-videoplayer-channel/README.md

```diff
@@ -1 +1,21 @@
+# Roku Sample Channel
 
+This Roku channel project demonstrates the usage of the Adobe Roku SDK.
+
+To run the project on your Roku device, follow the steps below to set it up beforehand.
+
+## Install the Roku SDK
+
+There are two options for installing the Roku SDK in this sample project:
+
+- Copy the SDK files to this sample project by following the [install instruction](../../Documentation/getting-started.md).
+
+- If you downloaded the entire GitHub repository, link the SDK source files to this project by running `make link-sdk` in the current directory.
```

Comment:
Can we avoid this by attaching and not linking the latest code to the sample apps? Same as we discussed for sample app with our release candidate.